playbook/app/views/layouts/playbook/samples.html.erb

```diff
@@ -8,17 +8,23 @@
         <%= stylesheet_packs_with_chunks_tag 'main', 'samples/index' %>
     </head>
     <body>
-        <div class="sample-layout">
-            <%= yield %>
-        </div>
+      <%= render 'layouts/playbook/nav'%>
+      <%= render 'layouts/playbook/mobile_hamburger'%>
+      <%= pb_rails("layout/sidebar", props: { classname: "pb--page--sideNav non-kit"}) do %>
+        <%= render 'layouts/playbook/sidebar'%>
+      <% end %>
+      <div class="sample-layout">
+        <%= yield %>
+      </div>
+      <%= render 'layouts/playbook/footer'%>
     </body>
 
     <%= javascript_include_tag "https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.js" %>
 
     <% if @type == 'react' %>
-      <%= javascript_packs_with_chunks_tag 'application', 'popover_docs', 'samples', 'full-page-samples' %>
+      <%= javascript_packs_with_chunks_tag 'application', 'popover_docs', 'samples', 'sample-show' %>
     <% else %>
-      <%= javascript_packs_with_chunks_tag 'application', 'rails-examples', 'samples', 'full-page-samples' %>
+      <%= javascript_packs_with_chunks_tag 'application', 'rails-examples', 'samples', 'sample-show' %>
```

Comment:
At some point, I want to make this list of pack tags into an instance variable within the controller so that we can keep it DRYer.